projects/wave-core-new/src/lib/map/map-layer.component.ts

```diff
@@ -299,7 +299,7 @@ export class OlRasterLayerComponent
         }
 
         this.source = new OlTileWmsSource({
-            url: this.backend.wmsUrl,
+            url: `${this.backend.wmsUrl}/${this.workflow}`,
```

Comment:
Should we rename it to *wmsBaseUrl* since we cannot access it without suffix anymore?